docs/source/data/fetching-rest.mdx

```diff
@@ -157,55 +185,67 @@ const { url } = await startStandaloneServer(server, {
 });
 ```
 
-If your `RESTDataSource` subclass accepts multiple arguments, make sure you add a constructor function, like so:
+</MultiCodeBlock>
+
+Passing the same `cache` to multiple `RESTDataSource` subclass instances enables those instances to _share_ the cached results.
+
+When running _multiple instances_ of your server, you should use an external shared cache backend. This enables one server instance to use the cached result from _another_ instance.
+
+> If you want to configure or replace Apollo Server's default cache, see [Configuring external caching](../performance/cache-backends) for more details.
+
+### `GET` requests and responses
```

Comment:
I like this! Tweaked it slightly and put it as a call out right below where we finish explaining how the caching works:

> (from the previous paragraph)If any resolver in the current operation attempts a parallel `GET` request to the same URL, `RESTDataSource` checks its memoized cache _before_ performing that request. If a request _or_ a result exists in the cache, `RESTDataSource` returns (or waits to return) that stored result **without** making another request.
>
> > This internal caching mechanism is why we create a new `RESTDataSource` instance for _every_ request. Otherwise, responses would be cached across requests even if they specify they shouldn't be!